rules/integrations/github/persistence_github_org_owner_added.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2023/09/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
```

Comment:
```suggestion
```

Reply:
@brokensound77 no longer required?

Reply:
@brokensound77 minimum stack version is still required because of `test_build_fields_min_stack` unit test. Technically we do not need to check anymore since our minimum backport is to 8.3 branch. we could adjust this test in a separate PR to default to the current branch, thus when backporting it would pass with no min stack defined but anything past 8.2 would fail as expected.

I wouldn't let this be a blocker @imays11.